### ISSUE TITLE
Can't install Cheat Engine

### ISSUE THREAD

**Issue by Mangaia** [cheat-engine/cheat-engine#2438]
Hello,

I'm unable to install Cheat Engine. The Setup just doesn't launch. I tried disabling all antivirus settings and realtime protections and etc, but nothing works! Can someone please help!

**Comment by cheat-engine** [comment]
Tried both the normal and lite version?

If neither works, then you may have an anti cheat installed on your system, or something else that doesn't want cheat engine to run.
do a `driverquery /verbose` and check for anti cheat drivers

**Comment by Mangaia** [comment]
I tried both the normal and lite version. I had League Of Legends installed and I uninstalled it. Still doesn't work. Nothing came up with the driverquery, couldn't see any anti cheat drivers.. I also did add exclusions to windows the /temp folder and the folder that the installer is in. Still nothing.

**Comment by phpjunkie420** [comment]
When I try to install Cheat Engine I had to select the file for exclusion, and the destination folder. After i do that I had no problems with install it. Might try disabling Windows Defender, then install Cheat Engine. Exclude the folder, and re-enable Windows Defender.

**Comment by Mangaia** [comment]
I did disable and excluded it, but still nothing... still gives me an error

**Comment by phpjunkie420** [comment]
So, when you double click the Cheat Engine icon to launch it nothing is happening? Do you have a virtual machine like VirtualBox?
Could try installing cheat engine on a virtual machine so you can get a look at all the files. See if everything is there on your PC. If everything is there then something is messing with the execution, or the installation.

I know this is a bit complicated by when you install programs there is stuff put in several different places. Like %USERPROFILE%\AppData, %PROGRAMDATA%, and the registry HKEY_CURRENT_USER\Software\Cheat Engine. Could try checking all of those places and doing a complete wipe of Cheat Engine, and reinstall it.

**Comment by Mangaia** [comment]
There is nothing about cheat engine in those places. When I run it, it shows the load bar, then after like 5-10 mins it loads up the installer and when I click next it loads up some stuff and ends up at this![image](https://user-images.githubusercontent.com/21020827/210363057-1cb6e316-9e36-4455-837d-3c4276fa9591.jpeg)

**Comment by fade2gray** [comment]
Sounds like you are disconnecting from the internet before running the installer.

**Comment by Mangaia** [comment]
How so? My internet is stable

**Comment by phpjunkie420** [comment]
Okay, try creating the folder for Cheat Engine first and exclude it from Windows Defender, and the installation file. Then try installing it.

This will take time but if you have a key to windows you can try installing it on virtual machine.

**Comment by Mangaia** [comment]
I'm not able to reach the point in which I select a place to install it at.. might try the virtual one in ab 10 mins and Ill u[date

**Comment by phpjunkie420** [comment]
https://www.virtualbox.org/ it's free.

**Comment by phpjunkie420** [comment]
It sounds like something in the background is intercepting and blocking the installation.

**Comment by fade2gray** [comment]
Are you using a VPN - you may have to disable it to complete installation.

**Comment by fade2gray** [comment]
See #2099

**Comment by fade2gray** [comment]
https://github.com/cheat-engine/cheat-engine/issues/2292#issuecomment-1200402172

**Comment by Mangaia** [comment]
None of these happen. Before I re-installed my PC I had Cheat Engine and it was working without any issues.

**Comment by Mangaia** [comment]
I'm unable to use Virtual Boxes apparently. So that won't do

**Comment by Mangaia** [comment]
Nope no 3rd parties. I tried disabling it, that didn't work. I made an online windows servers, installed it and transferred the installation folder and now it works. But BRUH, why doesn't it just install normally....

**Comment by fade2gray** [comment]
It really does sound like your ISP is the problem.

**Comment by Mangaia** [comment]
But how did it work like 10 days ago and now it doesn't? Really doesn't make sense. Also our ISPs here do not block anything LOL

**Comment by Mangaia** [comment]
Not even torrent downloading xD

**Comment by cheat-engine** [comment]
can you post your driverlist here ? Perhaps you have overlooked one anti cheat driver with a non-normal name

(`driverquery /verbose > filename`) and post the filename here

**Comment by Mangaia** [comment]
The command you gave, didn't work. So I did only driverquery /v  and this is the output : https://pastebin.com/KtCd8cH1

Thanks!

**Comment by cheat-engine** [comment]
i don't see any anti cheats directly, but fvevol.sys is suspicious

fvelol.sys is known to be a bitlocker file, but internally it's assiciated service is named gameflt which is a xbox service driver.

so we have a driver here that's mimicking 2 microsoft files, but messing up which it actually is.

try inspecting thay driver and check the signature and other details

**Comment by Mangaia** [comment]
> can you post your driverlist here ? Perhaps you have overlooked one anti cheat driver with a non-normal name
> 
> (`driverquery /verbose > filename`) and post the filename here

Anything out of the pastebin I sent?

**Comment by cheat-engine** [comment]
nothing i can see. But it's possible it hides itself from querying in the driverlist

Did you restart your computer after disabling your AV ? (not shut down and back on, else the av would still be blocking)

the copied files from the onlibe windows server do work when executing?

**Comment by Mangaia** [comment]
Yep, I did all that. Not shutdown, but actual restart. Nothing happens. I don't have an AntiVirus. All I got is windows defender. Which was fully disabled + firewall shutdown. When I installed on a Virtual Windows I copied the folder and it is now working fine. But installing it, doesn't.

**Comment by fade2gray** [comment]
> Before I re-installed my PC I had Cheat Engine and it was working without any issues.

When you re-installed Windows, was it a clean install or was it a repair install? If it was a repair install, I wonder if there are any old CE temporary installation files left in your temp directory - `C:\Users\<username>\AppData\Local\Temp\` - that could be interfering.

They will all start with `is-` and will look something like this...

![TempInstallerFiles](https://user-images.githubusercontent.com/1332009/211019225-92975e87-84db-45a7-9b0b-b8116e76136b.PNG)

If you find any, delete them all and try again.

**Comment by asdsadsaasd12312132** [comment]
I'm having the same issue. I disabled all anti-virus and firewalls.
